features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -315,7 +324,8 @@ Feature: Collection
     And I fill in "challenge_signup_offers_attributes_1_tag_set_attributes_character_tagnames" with "Teyla Emmagan, John Sheppard"
     And I press "Submit"
   Then I should see "Sign-up was successfully created"
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.